staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -163,9 +168,9 @@ func LoadEncryptionConfig(filepath string, reload bool, stopCh <-chan struct{})
 	}, err
 }
 
-func getTransformerOverridesAndKMSPluginHealthzCheckers(config *apiserverconfig.EncryptionConfiguration, stopCh <-chan struct{}) (map[schema.GroupResource]value.Transformer, []healthz.HealthChecker, *kmsState, error) {
+func getTransformerOverridesAndKMSPluginHealthzCheckers(ctx context.Context, config *apiserverconfig.EncryptionConfiguration) (map[schema.GroupResource]value.Transformer, []healthz.HealthChecker, *kmsState, error) {
```

Comment:
on all the methods where we're passing context, can we document the lifetime of the returned objects is tied to the context?

Reply:
@liggitt fixed, PTAL.